src/controllers/mentionEachUser.ts

```diff
@@ -23,34 +23,36 @@ export async function mentionEachUser(
 ) {
   const getMembersInServerResponse = await getMembersInServer(env);
   const roleId = transformedArgument.roleToBeTaggedObj.value;
-  const msgToBeSent = transformedArgument?.displayMessageObj?.value;
+  const msgToBeSent = transformedArgument?.displayMessageObj?.value; // Get custom message
   const dev = transformedArgument?.dev?.value || false;
-  // optional chaining here only because display message obj is optional argument
+
   const usersWithMatchingRole = filterUserByRoles(
     getMembersInServerResponse as UserArray[],
     roleId
   );
-  const payload = {
-    channelId: transformedArgument.channelId,
-    roleId: roleId,
-    message: msgToBeSent,
-    usersWithMatchingRole,
-  };
+
+  // Use the custom message if provided, otherwise construct the default message
+  let responseMessage;
+  if (usersWithMatchingRole.length === 0) {
+    responseMessage = msgToBeSent || "Sorry no user found under this role."; // Use custom message if available
```

Comment:
Here also show which role doesn't have users

Reply:
I have made the changes and it now shows the roles mentioned